static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -72,7 +59,7 @@ export const addPrivacySettingsLink = (): void => {
             );
 
             newPrivacyLink.addEventListener('click', () => {
```

Comment:
i think the callback here can now just be `cmp.showPrivacyManager`? the whole `initUi`/`show` thing can probably go?

Reply:
Yes, that's a good point. I've done the following in 5d9df00:

- remove the `show` export
- keep`consentManagementPlatformUi.show`
- replace calls sign-in gate `helper.js`